### PR TITLE
moved exercise 6 to exercise 1

### PR DESCRIPTION
**Pull request by tourunen**
- refactored all exercise numbers
- added debugging hints to exercise 5

### CODE REVIEW

File: exercises/ex1/README.md [hunk 1]
```diff
@@ -1,97 +1,104 @@
...
-   correctly with the `oc logs` command:
-   ```bash
-   oc logs <name of a pod>
-   ```
+    Check all the objects that were created by the single command
+    ```bash
+    oc get all
+    ```
+
+    We then expose it over HTTPS by creating a **Route**
```
Comment (rlaurika) [inline exercises/ex1/README.md]: We could link to the documentation for Routes at the top of the document as well in this exercise.

File: exercises/ex1/ex1-arch.svg [hunk 2]
```diff
@@ -1,236 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (rlaurika) [inline exercises/ex1/ex1-arch.svg]: I think if you use "git mv" these might show up differently? I guess it doesn't detect properly that the files were moved, which messes up the history?

Comment (rlaurika) [reply exercises/ex1/ex1-arch.svg]: You end up with one "deleted" file and one "new" file unless you use git mv is what I'm trying to say.

**Comment by tourunen** [comment]
Let's have another go with 'git mv' as a separate step

**Pull request closed without merging**